src/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -19,13 +20,15 @@ import { useMutation } from "Utils/Hooks/useMutation"
 import { useAuthDialog } from "Components/AuthDialog"
 
 interface FollowArtistButtonProps extends Omit<ButtonProps, "variant"> {
+  me: FollowArtistButton_me$data | null
   artist: FollowArtistButton_artist$data
   contextModule?: AuthContextModule
   triggerSuggestions?: boolean
   onFollow?: (followed: boolean) => void
 }
 
 const FollowArtistButton: React.FC<FollowArtistButtonProps> = ({
+  me,
```

Comment:
The changes to this allow us to listen to this field elsewhere and detect when we've actually followed to trigger dismissal.